easybuild/easyblocks/p/pytorch.py

```diff
@@ -475,13 +523,62 @@ def _set_cache_dir(self):
         mkdir(cache_dir, parents=True)
         env.setvar('XDG_CACHE_HOME', cache_dir)
 
+    def _compare_test_results(self, old_result, xml_result, old_failed_test_names, xml_failed_test_names):
+        """Compare test results parsed from stdout and XML files"""
+        diffs = []
+
+        old_suite_names = {suite.name for suite in old_result.failed_suites}
+        new_suite_names = {suite.name for suite in xml_result.failed_suites}
+        new_suites = new_suite_names - old_suite_names
+        missing_suites = old_suite_names - new_suite_names
+
+        diffs = []
+        if new_suites:
+            diffs.append(f'Found {len(new_suites)} new suites in XML files: {", ".join(sorted(new_suites))}')
+        if missing_suites:
+            diffs.append(f'Did not found {len(missing_suites)} suites in XML files: ' +
```

Comment:
Do you mean "Did not find" or "Did find"

Reply:
Yes, just a typo. "not find". Fixed